src/google-classroom-extractor/tests/api/api_caller/test_api_caller.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Why is this file commented?

Reply:
Removed - it's part of the API testing i was working on.